TCAT/AppDelegate.swift

```diff
@@ -92,6 +93,18 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                 Analytics.shared.log(payload)
             }
         }
+        
+        // v1.4.1 Delete Corrupted Shortcut Donations
```

Comment:
Can you just put all the patch functions in a `patchFunctions()` wrapper and get it out of that first AppDel function?

Reply:
I tried doing that, but I wasn't sure how to pass the `rootVC` that the first path (v1.3 Data Migration) needs

Reply:
Oo right. You can pass it as a parameter maybe?

Reply:
I'll try out passing it as a parameter later today -- if it gets complicated I think it'd be best just to leave both in there